components/x-follow-button/stories/index.js

```diff
@@ -0,0 +1,15 @@
+const { FollowButton } = require('../');
+
+module.exports = {
+	component: FollowButton,
+	dependencies: {
+		'o-fonts': '^3.0.0',
+		'o-typography': '^5.5.0'
+	},
+	stories: [
+		require('./with-concept-name'),
+		require('./without-concept-name')
```

Comment:
It might be nicer to have a single story and have `conceptNameAsButtonText` as a knob, so you can turn it on and off without switching between stories. Should also result in less code too.

Reply:
👍